### PR TITLE
add shared-workarea

### PR DESCRIPTION
**Pull request by athornton**

### CODE REVIEW

**Review by rra — APPROVED** [review]

File: services/portal/values-idfprod.yaml [hunk 1]
```diff
@@ -3,8 +3,8 @@ firefly:
   replicaCount: 4
 
   volumes:
-    firefly_workarea_nfs:
-      path: /share1/home/firefly/workarea
+    firefly_shared_workarea_nfs:
+      path: /share1/home/firefly/shared-workarea
```
Comment (gpdf) [inline services/portal/values-idfprod.yaml:7]: Here you're using Google Cloud resources to create a persistent NFS filesystem that's visible from all the pods, under that pathname, right?

File: services/portal/values-int.yaml [hunk 2]
```diff
@@ -47,7 +47,7 @@ firefly:
     runAsGroup: 102
 
   volumes:
-    firefly_workarea_hostpath: /sui/firefly/workarea
+    firefly_shared_workarea_hostpath: /sui/firefly/workarea
```
Comment (gpdf) [inline services/portal/values-int.yaml:50]: This instance of `workarea` on the RHS is part of the "real" GPFS pathname of the filesystem - something that already physically exists on GPFS at NCSA and therefore (at least for now) should not be changed.  Is that correct?

Same for `values-stable.yaml` below.

**Review by gpdf — COMMENTED** [review]
Looks reasonable to me

File: services/portal/values-int.yaml (hunk 2 above)
Comment (athornton) [reply services/portal/values-int.yaml:50]: That is correct.  We would need sysadmin assistance at NCSA to change it...so let's not, at the moment.

File: services/portal/values-idfprod.yaml (hunk 1 above)
Comment (athornton) [reply services/portal/values-idfprod.yaml:7]: Correct.  I did that by hand, but we will want to roll it into our FS provisioning automation.

**Pull request merged**